internal/cmd/deployorder.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/simplesurance/dependencies-tool/internal/deps"
+)
+
+const deployOrderShortHelp = "Generate a deployment order."
+
+var descrDependencyFileNames = strings.TrimSpace(`
+Dependency definition files are discovered by searching in all child directories
+of PATH.
+Files that match the following names are discovered, only the first found per
+directory is parsed, the preference order is:
+
+  1. .deps-<ENVIRONMENT>-<REGION>.toml
+  2. .deps-<ENVIRONMENT>.toml
+  2. .deps.toml
+`)
+
+var deployOrderLongHelp = deployOrderShortHelp + "\n\n" + strings.TrimSpace(`
+Positional Arguments:
+  PATH		- Root Directory for the dependency file discovery.
```

Comment:
PATH currently could also be a parameter with a default of `.`.
In a follow-up PATH will also be able to accept a path to marshaled dependency tree file.
Specifying the repository or marshal file path will then be required. 
A positional argument will bit better that future use case.